src/bot.py

```diff
@@ -1,40 +1,119 @@
+
         #Configure bot
             #TODO set from configuration, any intents we might need. Can also set contexts in configuration and do it that way
-        intents = discord.Intents.default() 
+        intents = discord.Intents(messages=True, guilds=True, members=True, reactions=True) 
             #TODO set description from configuration
         description = '''A placeholder bot description.'''
 
         self.bot = commands.Bot(command_prefix='!', description=description, intents=intents)
 
+        bw = open('/opt/bits/src/bWords.txt')
+        bwlines = bw.readlines()
+        b_words = []
+        for i in bwlines:
+            b_words.append(i[:-1])
+
+        #Logs the bot starting up
         @self.bot.event
         async def on_ready():
             self.log.info(f'Logged in as {self.bot.user} (ID: {self.bot.user.id})')
+            print('Bot has been started')
+            global guild
+            guild = self.bot.get_guild(self.config.data['bot_settings']['guildID'])
+            self.log.info('Guild has been set to: ' + str(guild.name) + ' | ' + str(guild.id))
+         
+        @self.bot.event
+        async def on_message(message):
+            if message.author == self.bot.user:
+                return
+
+            msg = message.content.lower().replace(" ","")
 
-        @self.bot.command() #TODO Break this out into a class or some other logical structure not in Bot.init
+            if any(word in msg for word in b_words):
+                vio_resp = "Hault {}, you have violated the law. you dirty slut. keep swearing and I will kill you, you little shit".format(message.author)
+                await message.channel.send(vio_resp)
+                await message.delete()
+            await self.bot.process_commands(message)
+                
+        #Member_check is used for commands run outside of Guild channels (ie DMs). Built-in has_role only works in Guild channels
+        def member_check(ctx):
+            member = guild.get_member(ctx.author.id)
+            if member == "None":
+                self.log.info(f'{ctx.author.name}:{ctx.author.id} exectued {ctx.invoked_with} and is NOT a member')
+                return False
+            else:
+                newb_role = guild.get_role(newb_check)
+                return member.top_role != newb_role
+
+        #Verifies bot is online
+        @commands.has_role(int(admin_check))
+        @self.bot.command()
         async def alive(ctx):
-            """Check daemon status from server."""
-            self.log.info(f"[{ctx.author.id}:{ctx.author}] executed command '{ctx.invoked_with}'")
-            
-            
-            #TODO This should be decorated as admin only
+            self.log.info(f"{ctx.author.id}:{ctx.author} executed command '{ctx.invoked_with}'")
+            await alive_command(ctx, self)
+            return
+
+        #Returns Username and Top Role
+        @commands.has_any_role(int(mod_check), int(admin_check))
```

Comment:
This decorator is a built-in check in the Discord API. The mod_check and admin_check are the role IDs that are checked in the server. 
I'll probably replace with something more like how I wrote member_check anyway since that works when commands are executed in DMs whereas .has_any_role does not